### ISSUE TITLE
-a C not aligning image to the center

### ISSUE THREAD

**Issue by Kerooker** [petrkutalek/png2pos#11]
The expected behaviour is the image being aligned to the center, but it is still aligned left.

Reproducing in my machine:

Executing `png2pos -o <resultPath> -a C <imagePath>`

Image being tested (magePath):
http://i.imgur.com/6Cj6C8O.png

ResultPath bytes being sent directly to the POS Printer

**Comment by petrkutalek** [comment]
@Kerooker Unfortunately I could not reproduce your problem.
```sh
$ wget http://i.imgur.com/6Cj6C8O.png
$ make
$ ./png2pos -V
png2pos 1.6.22 (Jan 24 2017)
LodePNG 20161127
$ ./png2pos -o /dev/usb/lp3 -a C 6Cj6C8O.png
```
This produced correct output: 

![debug](https://cloud.githubusercontent.com/assets/7659931/22259883/546e4a34-e267-11e6-8ed2-c9ef3d50a5a1.png)

Please, could you provide me with the following information?
- png2pos version (`png2pos -V`)
- produced binary file (feel free to sent it via email to petr@kutalek.cz)
- model of your printer
- your png2pos related env variables (`set | grep PNG2POS`)

**Comment by Kerooker** [comment]
Version 1.6.19
Binary: https://mega.nz/#!oB8GWKBL!4Fn8p-eTsRmN7s1LMUrJZgs1ILApPWP5BcXPP9wtMwQ
Model: HPRT-PPT2A (https://www.hprt.com/Product/POS-Printer/PPT2-A.html)
Env variables: PNG2POS_PRINTER_MAX_WIDTH = 384

Produces:
(First one is left aligned, second one is center aligned with -a C, sorry for the wannabe-quality)

![image](https://cloud.githubusercontent.com/assets/1577251/22320026/87228e1e-e36f-11e6-8016-06bc37a62432.png)

**Comment by petrkutalek** [comment]
In some version after 1.6.20 I have changed the way `png2pos` aligns images (Now it does not depend on ESC/POS commands supported by printer). Please, could you check the latest version? If you use Windows, are you able to build it yourself?

**Comment by Kerooker** [comment]
I'm not able to build it myself, currently tracking an error in SO http://stackoverflow.com/questions/41877378/mingw32-make-command-unrecognized-option-z

**Comment by Kerooker** [comment]
Update:
I was able to build in windows (thanks to SO), and the result is still the same: The image is not centered. 

Info:
Version: 1.6.22

Binary: https://mega.nz/#!kQcWjKjL!-8GsAOqmHicYWaPDN71kPkvOLJUjr-pv-TWgkm7xD_Y

Model: HPRT-PPT2A (https://www.hprt.com/Product/POS-Printer/PPT2-A.html)

Env variables: PNG2POS_PRINTER_MAX_WIDTH = 384, tried with and without it

Produces the same output as the image above.

**Comment by petrkutalek** [comment]
I have built it for you [Windows, v1.6.22](https://github.com/petrkutalek/png2pos/releases/tag/v1.6.22). Sorry, I probably did not make myself clear last post. I would need binary file produced by `png2pos` via `-o` command line parameter.

**Comment by Kerooker** [comment]
Oh, yeah, I might have misunderstood that.

File generated via -o
https://mega.nz/#!cJ8CASDS!yWccfBzBVFv6mN2_qslgSAZ-CwnNznCyACR8VK0lCJk

**Comment by petrkutalek** [comment]
@Kerooker, look:

1) I have tested `png2pos.exe` I have built for you, it is OK. Have you tried it?
2) I have sent your `result6319277513862944684.tmp` file to my printer and this is correct result:

![20170126_194851](https://cloud.githubusercontent.com/assets/7659931/22345588/f7bc167c-e400-11e6-91bf-c5f3afa34e38.jpg)

![w](https://cloud.githubusercontent.com/assets/7659931/22345603/035a72f8-e401-11e6-9722-c717d51903fe.png)

`png2pos` centers image correctly, it does not depend on your printer (`png2pos` itself adds white border around). I have no idea what's going on and I could not help you in any other way, I am sorry.
